app/interactors/schedule/create_interactor.rb

```diff
@@ -45,4 +46,12 @@ def schedule_to_publish
     reviewed = params[:review_status] == "reviewed"
     ScheduleService.new(edition).schedule(user: user, reviewed: reviewed)
   end
+
+  def create_timeline_entry
+    TimelineEntry.create_for_status_change(
+      entry_type: :schedule_updated,
```

Comment:
I think this should be `:scheduled`

Reply:
Shame we're not catching that on a test - if I'm right.

Reply:
Oops - good spot! This is an unfortunate side-effect of the same copy being used in the scheduling notice. I've amended the test to catch it now.